### ISSUE TITLE
[studio] dependency resolver config does not pick up gif files

### ISSUE THREAD

**Issue by russdanner** [craftercms/craftercms#2314]
### Expected behavior
Gifs should be identified by dependency resolver as an asset

### Actual behavior
Regex does not catch gif images.  Other image types work

### Steps to reproduce the problem
* create a new site with empty blueprint 
* edit home page, in the RTE add a png image
* check dependencies for home page, see referenced, png is there
* edit home page, in the RTE add a gif image
* check dependencies for home page, see referenced, png is NOT there

### Work around
Add gif regex to resolver for pages and components
```
                <dependency-type>
                    <name>asset</name>
                    <includes>
                        <!-- path patterns to look for (multiple) -->
                        <pattern>
                            <find-regex>/static-assets/([^&lt;"'\)]+)</find-regex>
                        </pattern>
                        <pattern>
                            <find-regex>/static-assets/([^&lt;"]+)\.gif</find-regex>
                        </pattern>
                    </includes>
                </dependency-type>

```
### Log/stack trace (use https://gist.github.com)
N/A

#### Version
Studio Version Number: 3.0.16-SNAPSHOT-96bdf1
Build Number: 96bdf1b8e4da646d6be9f5eca8ffd20ef1396ab1
Build Date/Time: 07-31-2018 14:02:27 -0400

Applies to 2.5.x as well

#### OS
Any

#### Browser
Any

**Comment by dejan-brkic** [comment]
The workaround is the solution. This is a configuration issue and is resolved only by tweaking configuration to achieve goals. The only action item can be if additional regex is generic enough to be added as the out-of-the-box regex.

**Comment by russdanner** [comment]
@dejan-brkic based on your comment this should be assigned to someone to update the blueprints then?

**Comment by dejan-brkic** [comment]
If this is a regex that needs to be added to out-of-the-box config, then it should be assigned to someone to update blueprints.

I don't think this regex is generic enough to be added to blueprints. It is less restrictive regarding naming convention we have established with the first regex, and it is restricted to GIF only.

**Comment by russdanner** [comment]
Thanks @dejan-brkic I assigned this to you.  @alhambrav can update the blueprints but she needs help with the proper regex.

**Comment by sumerjabri** [comment]
@alhambrav please work with Dejan to update all BPs to have the right regex.

**Comment by alhambrav** [comment]
@dejan-brkic Hi!  I followed the steps listed in the ticket, and I also enabled debug logs, using an empty bp, I inserted a gif file, `giphy.gif` from the RTE then saved the changes.  I then clicked on dependencies, and the gif file is not listed.  So, I looked in the `dependency` table in the database, and the gif file is not there.  I looked at the logs, and for some reason, there are numbers after the gif file `/static-assets/item/images/2018/08/20/giphy.gif?1534788414265` when it was in the dependency resolver module? Here's the log:
